### PR TITLE
Feature/setup

### PR DESCRIPTION
**Pull request by owieth**

### CODE REVIEW

**Comment by ninuxch** [comment]
IMO good approach, foundry and hardhat.

File: STYLE.md [hunk 1]
```diff
@@ -0,0 +1,236 @@
+# Solidity Style Guide
```
Comment (ninuxch) [inline STYLE.md:1]: Fine with that.

Comment (ninuxch) [reply STYLE.md:1]: We should habe a styleguide for typescript as well. This is a monorepo no?

File: README.md [hunk 2]
```diff
@@ -38,4 +37,4 @@ Push to develop and hack your feature on feat/featname....
 - Tendderly Sim
```
Comment (ninuxch) [inline README.md:37]: Tenderly with one d

File: remappings.txt [hunk 3]
```diff
@@ -0,0 +1,2 @@
+@std/=lib/forge-std/src/
```
Comment (ninuxch) [inline remappings.txt:1]: Those abbreviations should go into the readme as they are configured per project, i.e. I'm using @openzeppelin instead of @oz

Comment (ninuxch) [reply remappings.txt:1]: Also we should mention that foundry requires git checkout with --recursive

**Review by ninuxch — APPROVED** [review]
Overall a good, actually excellent start imo.

File: STYLE.md (hunk 1 above)
Comment (owieth) [reply STYLE.md:1]: I'm not a big fan of monorepos cross langauge so the web interfaces should have their own repositories

**Pull request merged**